libs/dev-tools/src/lib/services/dr/create-dr-input.ts

```diff
@@ -30,6 +35,11 @@ export type PostDataRequestInput = {
 	 * Inputs encoded in bytes. Input depends on the Tally Oracle Program specified in tallyProgramId.
 	 */
 	tallyInputs: Uint8Array;
+	/**
+	 * Amount of gas units allowed for tally
+	 * Defaults to 300_000
+	 */
+	tallyGasLimit?: number;
```

Comment:
Not bigint?

Reply:
As discussed: `bigint` causes issues with the JSON serialisation due to the JS implementation. But as we're not supporting numbers larger than `300_000_000_000_000` for now we're leaving it as a number.